components/calendar/calendar.js

```diff
@@ -509,7 +552,7 @@ class Calendar extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 		};
 		const labelId = `${this._tableInfoId}-heading`;
 		const labelledBy = this._dialog ? labelId : undefined;
-		const heading = `${calendarData.descriptor.calendar.months.long[this._shownMonth]} ${this._shownYear}`;
+		const heading = formatDate(new Date(this._shownYear, this._shownMonth, 1), {format: 'monthYear'});
```

Comment:
I realized that I had just been formatting my own Month Year heading rather than using `formatDate` for this. A few visual diff screenshots will be updated to reflect this.